tools/scrimlet/softnpu-init.sh

```diff
@@ -13,27 +13,29 @@ echo "Using $GATEWAY_IP as gateway ip"
 GATEWAY_MAC=${GATEWAY_MAC:=$(arp "$GATEWAY_IP" | awk -F ' ' '{print $4}')}
 echo "Using $GATEWAY_MAC as gateway mac"
 
-# Sidecar Interface facing "sled"
-./out/softnpu/swadm -h "[fd00:1122:3344:101::2]" port create 1:0 100G RS
+SWADM="./out/softnpu/swadm -h [fd00:1122:3344:101::2]"
```

Comment:
Just a quick note on this. Based on @Nieuwejaar's comment [here](https://github.com/oxidecomputer/omicron/pull/2645#discussion_r1157531870), we're moving away from using an ad-hoc download of `swadm` and using the `swadm` located in the `oxz_switch` zone via `zlogin`. This is implemented in both #2645 and #2694 so depending on how this PR gets ordered in with those, we'll want to be careful on rebase/merge to preserve the `zlogin` based scheme.

Reply:
Rebased on changes that include #2694, thanks!